backend/project/urls.py

```diff
@@ -15,7 +15,11 @@
 """
 from django.contrib import admin
 from django.urls import path
+from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView
```

Comment:
I think it would be good to have the redoc too.

Reply:
I just added it!